### ISSUE TITLE
"The system detected an overrun of a stack-based buffer in this application" error

### ISSUE THREAD

**Issue by xyberdan** [teusink/Home-Security-by-W10-Hardening#27]
Which registry key create this error when someone want to configure OS and how to fix it?
This window shows after hardening.

---------------------------
Settings: SystemSettings.exe - System Error
---------------------------
The system detected an overrun of a stack-based buffer in this application. This overrun could potentially allow a malicious user to gain control of this application. 
---------------------------

**Comment by teusink** [comment]
Oof, not sure at the moment. I run it on all my Windows systems (various vendors) but no such error. But I am not on the Windows May 2020 update. Are you perhaps?

I did faced SystemsSettings.exe error myself, but that was due to corrupt files, and the error was different.

See these links:
https://answers.microsoft.com/en-us/windows/forum/all/solvederror-system-detected-an-overrun-of-a-stack/23ca258b-480a-448f-bd52-78591ea34cac
https://windowsreport.com/stack-based-buffer-application-overrun/

My advice, execute the following steps in this order.

- Open a CMD window in Administrator mode
- Run these commands:

** Clean up the image with unused stuff **
Dism.exe /online /Cleanup-Image /StartComponentCleanup
Dism.exe /online /Cleanup-Image /StartComponentCleanup /ResetBase

** Verify the image files that are used **
Dism.exe /online /Cleanup-Image /RestoreHealth

** Verify system installation with the verified image **
sfc /scannow

And then do a reboot.
